napari/util/misc.py

```diff
@@ -129,7 +130,9 @@ def compute_max_shape(shapes, max_dims=None):
     if max_dims is None:
         max_dims = max(len(shape) for shape in shapes)
 
-    max_shape = [0, ] * max_dims
+    max_shape = [
+        0,
+    ] * max_dims
 
```

Comment:
To me this looks awkward.

Reply:
This is a feature – lists, argument lists, etc. will be split apart in the same style as `black` when the last element ends with a comma.

Reply:
Yeah not a fan, feature or not. In this specific case it's *much* worse than the one-line version, because it is conceptually a one-line thing.